README.md

```diff
@@ -15,12 +15,11 @@ $ pip3 install ak-phrase.py
 ## Usage
 
 ```python
-from sentence_generator import SentenceGenerator
+import ak-phase.py
```

Comment:
I just checked.
I have added it wrong in the issue.
It should be like this `from ak-phrase.py import SentenceGenerator`